### PR TITLE
Improve the bootstrap output

### PR DESCRIPTION
**Pull request by gaganhegde**
**What type of PR is this?**
> /kind cleanup

**What does does this PR do / why we need it**:
Lists out the high-level process of events that occur during the bootstrap process.

See https://github.com/rhd-gitops-example/doc

**Which issue(s) this PR fixes**:
https://issues.redhat.com/browse/GITOPS-211
Fixes #?

### CODE REVIEW

File: pkg/pipelines/bootstrap.go [hunk 1]
```diff
@@ -472,7 +474,9 @@ func createCICDResources(fs afero.Fs, repo scm.Repository, pipelineConfig *confi
 			return nil, err
 		}
 		outputs[dockerConfigPath] = dockerSecret
+		log.Success("The secrets have been encrypted")
 		outputs[serviceAccountPath] = roles.AddSecretToSA(sa, dockerSecretName)
+		log.Success("Service account Role-binding added to secrets")
```
Comment (bigkevmcd) [inline pkg/pipelines/bootstrap.go:479]: I think this is a bit too low-level.

File: pkg/pipelines/bootstrap.go [hunk 2]
```diff
@@ -157,6 +158,7 @@ func Bootstrap(o *BootstrapOptions, appFs afero.Fs) error {
 	if err != nil {
 		return fmt.Errorf("failed to build resources: %v", err)
 	}
+	log.Success("The default dev/stage environment have been created")
```
Comment (bigkevmcd) [inline pkg/pipelines/bootstrap.go:161]: How about "Created the following environments: cicd, dev, stage" but with the namespaces for these environments?

File: pkg/pipelines/bootstrap.go [hunk 3]
```diff
@@ -483,6 +487,7 @@ func createCICDResources(fs afero.Fs, repo scm.Repository, pipelineConfig *confi
 			return nil, err
 		}
 		outputs = res.Merge(outputs, trackerResources)
+		log.Success("Optional Pipelines tracker has been configured")
```
Comment (bigkevmcd) [inline pkg/pipelines/bootstrap.go:490]: We can drop "Optional" here, as it is implied by the fact that we asked the user if they wanted it (or they explicitly configured it).

File: pkg/pipelines/bootstrap.go [hunk 4]
```diff
@@ -499,11 +504,13 @@ func createCICDResources(fs afero.Fs, repo scm.Repository, pipelineConfig *confi
 	outputs[pushTemplatePath] = triggers.CreateCIDryRunTemplate(cicdNamespace, saName)
 	outputs[appCIPushTemplatePath] = triggers.CreateDevCIBuildPRTemplate(cicdNamespace, saName)
 	outputs[eventListenerPath] = eventlisteners.Generate(repo, cicdNamespace, saName, eventlisteners.GitOpsWebhookSecret)
+	log.Success("OpenShift Pipelines resources created")
 	route, err := routes.Generate(cicdNamespace)
 	if err != nil {
 		return nil, err
 	}
 	outputs[routePath] = route
+	log.Success("OpenShift Route has been configured")
```
Comment (bigkevmcd) [inline pkg/pipelines/bootstrap.go:513]:
```suggestion
	log.Success("OpenShift Route for the EventListener created")
```

File: pkg/pipelines/bootstrap.go [hunk 5]
```diff
@@ -472,7 +474,9 @@ func createCICDResources(fs afero.Fs, repo scm.Repository, pipelineConfig *confi
 			return nil, err
 		}
 		outputs[dockerConfigPath] = dockerSecret
+		log.Success("The secrets have been encrypted")
```
Comment (bigkevmcd) [inline pkg/pipelines/bootstrap.go:477]: How about "Authentication tokens encrypted in secrets" ?

**Review by bigkevmcd — CHANGES_REQUESTED** [review]

**Comment by wtam2018** [comment]
@gaganhegde  We should probably do this PR in the new cli repo instead https://github.com/rhd-gitops-example/gitops-cli

**Pull request closed without merging**